<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsPageWidget</class>
 <widget class="QWidget" name="ContactsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <item>
      <widget class="QWidget" name="friendWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="personButton">
            <property name="text">
             <string>好友</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src/icon/view_off.svg</normaloff>
              <normalon>src/icon/view.svg</normalon>src/icon/view_off.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ContactsListWidget" name="personList"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="groupWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="groupButton">
            <property name="text">
             <string>群组</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src/icon/view_off.svg</normaloff>
              <normalon>src/icon/view.svg</normalon>src/icon/view_off.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ContactsListWidget" name="groupsList"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactsListWidget</class>
   <extends>QListWidget</extends>
   <header>ContactsListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
